testpilot/frontend/templates/testpilot/frontend/index.html

```diff
@@ -25,7 +25,6 @@
     })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
 
     ga('create', 'UA-49796218-34', 'auto');
```

Comment:
Threw together this quick PR because I was fast on the merge button:

https://github.com/mozilla/testpilot/pull/587